react-app/src/stylesheets/Page.scss

```diff
@@ -75,3 +90,27 @@
   flex-direction: column;
   justify-content: center;
 }
+
+.file-picker-box {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  vertical-align: center;
+}
+
+.file-button-container {
+  display: flex;
+  justify-content: space-betweeen;
+}
+
+.file-picker {
+  font-size: 14px;
+  margin-bottom: 0px;
+  margin-left: 20px;
```

Comment:
DP3 - This style is used on more than just the file-picker button; it's also used on the delete button. Could we rename the class to be something more appropriate? Also I'll highlight again the utility classes in the Bootstrap cheatsheet that allow for easy inline margin-manipulation (see the `mb-x` classes [here](https://hackerthemes.com/bootstrap-cheatsheet/)). Those may be of use in lieu of a specific class. Up to you.